docs/articles/roadmap/README.md

```diff
@@ -15,28 +16,30 @@ Please vote on the features you want us to implement first to help us prioritize
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | **In Progress**
-[API Testing](https://github.com/DevExpress/testcafe/issues/4288)                                 | Planned
+[HTTP API Testing](https://github.com/DevExpress/testcafe/issues/4288)                            | Planned
+[HTTP2/HTTP3 Support](https://github.com/DevExpress/testcafe-hammerhead/issues/1960)              | Planned
+[Set HTTP Cookies](https://github.com/DevExpress/testcafe/issues/4063)                            | Planned
 
 ## API
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Configuration File](https://github.com/DevExpress/testcafe/issues/3131)                          | **Implemented**
-[Screenshots of the Entire Page](https://github.com/DevExpress/testcafe/issues/1520)              | **Implemented**
-[Browser Information in Test Code](https://github.com/DevExpress/testcafe/issues/481)             | **Implemented**
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | **Implemented**
+[Dedicated *Scroll* Action](https://github.com/DevExpress/testcafe/issues/527)                    | Planned
+[View Active Cookies](https://github.com/DevExpress/testcafe/issues/4428)                         | Planned
```

Comment:
It's not `active` cookie.

I propose the the `Inspect cookies` term.

Reply:
Ok, let's call it `Cookie management` if you don't mind

Reply:
ok, `Cooke Management` it is. Since there is no issue for both get and set, I've made a new one: [Cookie Management](https://github.com/DevExpress/testcafe/issues/5964)